plugins/module_utils/linode_common.py

```diff
@@ -107,6 +107,7 @@
 class LinodeModuleBase:
     """A base for all Linode resource modules."""
 
+    ## pylint: disable=too-many-positional-arguments
```

Comment:
It seems we've handled disabling this linter in different places/style: https://github.com/search?q=repo%3Alinode%2Fansible_linode%20too-many-positional-arguments&type=code Could you pull the latest change from dev to have them organized?